sklearn/model_selection/_validation.py

```diff
@@ -22,8 +22,8 @@
 
 from ..base import is_classifier, clone
 from ..utils import (indexable, check_random_state, _safe_indexing,
-                     _message_with_time)
-from ..utils.validation import _check_fit_params
+                     _message_with_time, safe_indexing)
```

Comment:
use `_safe_indexing` instead of `safe_indexing`, as we decided to remove the latter from the public API in preference for an internal-only utility.